data/metadata.json

```diff
@@ -499,6 +499,27 @@
     "hack-toolbox/lockscreen/inactive": {
         "sound-file": "common/lockscreenInactive.webm"
     },
+    "hack-toolbox/lockscreen/open/lock.fizzics.1": {
+        "sound-file": "toolbox/Lockscreen_fizzics1_open.webm"
+    },
+    "hack-toolbox/lockscreen/open/fizzics2a-currently-unused": {
```

Comment:
Or you could add  `"note": "currently unused"` as in https://github.com/endlessm/hack-sound-server/blob/ed256f27f93190dd31dd30fef92db176e82b0770/data/metadata.json#L553

But this is just a nitpick. Up to you to follow or not this. Then feel free to merge this.

Reply:
I like that idea better than what I had, I'll change it.